### PR TITLE
Preferences - Add Open Preferences Folder to Preferences save dialogues #1538

### PR DESCRIPTION
**Pull request by Draise14**
This had a chunk of messy commits, but now it's ok. I have tested it by delete/adding the appdata folder, and things are conditionally going to LOCAL or USER accordingly. 

The messy commits was due to leaving in comments, first conditional was not working, and the first commit for the idea wasn't quite right. 

This may need testing in Linux and Mac to make sure it's ok, but according to:
[This link](https://docs.blender.org/manual/en/2.79/getting_started/installing/configuration/directories.html), it shoooould be relative to the OS according to how Blender works internally with paths. 

### Note
The USER goes to parent folder, showing all bforartists preferences. Though this might not be necessary.

### CODE REVIEW

File: release/scripts/startup/bl_ui/space_userpref.py [hunk 1]
```diff
@@ -133,15 +136,15 @@ def draw(self, context):
         layout.operator_context = 'INVOKE_AREA'
         layout.operator("wm.read_factory_userpref", text="Load Factory Preferences")
 
-#work
+#work        
         layout.separator()
 
-        if os.path.isdir(Path(bpy.utils.resource_path('USER'))):
+        if os.path.isdir(Path(bpy.utils.resource_path('USER'))): 
             layout.operator("wm.path_open", text="Open Appdata Folder", icon = "FOLDER_REDIRECT").filepath = str(user_path)
```
Comment (Draise14) [inline release/scripts/startup/bl_ui/space_userpref.py:143]: "Open Preference Folder" might make more sense.

**Comment by ReinerBforartists** [comment]
Not so fast with the pull requests. First finish the relevant tasks ^^

Do we have the same menu at two locations now?

And what about Linux and Mac? Have you tested it?

**Comment by Draise14** [comment]
It's in one location. The old version in the first proposal was reverted. 

I don't have a linux build on hand, and my mac is decomissioned right now. I'm pending someone to test I suppose. I guess this testing process should be done in the task before pulling?

**Comment by Draise14** [comment]
I think I'll close this for now, till I've tested it. I've found a couple of linux users who can try out the script for me. Looking for a mac user now.

**Pull request closed without merging**